lib/squadster/workers/notify_attendants.ex

```diff
@@ -1,11 +1,39 @@
 defmodule Squadster.Workers.NotifyAttendants do
   use Task
 
+  import Ecto.Query
+  import HTTPoison
+
+  alias Squadster.Formations.SquadMember
+  alias Squadster.Repo
+
+  @bot_endpoint "https://mercury-dev.herokuapp.com/message"
```

Comment:
We should store it in config, and take from ENV variable. See how it implemented for `frontend_url` in `auth_controller`